eventsapi.py

```diff
@@ -1,42 +1,38 @@
 import datetime
 import requests
+import os
 
 # For more information, check out our support page
 # https://support.1password.com/events-reporting
 
-
-# Replace APITOKEN with your generated Events API token
-api_token = "Bearer APITOKEN"
-
-# Your URL corresponds to your 1Password account region
+api_token = "Bearer "+os.environ['EVENTS_API_TOKEN']
```

Comment:
An f-string here would be cool. Also maybe on the url strings below?

```suggestion
api_token = f"Bearer {os.environ['EVENTS_API_TOKEN']}"
```